lib/Hero/Content.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Button as ButtonComponent } from '../Button';
+
+const Content = ({
+  classNames = {},
+  title,
+  subTitle,
+  imageUrl,
+  hasOverlay,
+  NextImage,
+  Button = () => (
+    <ButtonComponent
+      displayText="Shop Now"
+      onClick={() => {
+        alert('You clicked me');
+      }}
+    />
+  ),
+}) => {
+  /**
+   * Next Image loader
+   * @returns {*}
+   */
+  const myLoader = () => {
+    return imageUrl;
+  };
+  return (
+    <>
+      <div
+        className={clsx('flex items-center justify-center', classNames.wrapper)}
+        style={{
+          background: NextImage === undefined ? 'url(' + imageUrl + ')' : '',
+          backgroundPosition: 'center',
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+          height: '100vh',
+          width: '100vw',
+        }}
+      >
+        {NextImage && (
```

Comment:
Same comment as before — let's refrain from calling this prop `NextImage` and let's use `Image` for a platform-agnostic approach.